libraries/typings-generator/src/TypingsGenerator.ts

```diff
@@ -88,27 +98,39 @@ export class TypingsGenerator {
   public async generateTypingsAsync(): Promise<void> {
     await FileSystem.ensureEmptyFolderAsync(this._options.generatedTsFolder);
 
-    const filePaths: string[] = glob.sync(path.join('**', `*+(${this._options.fileExtensions.join('|')})`), {
-      cwd: this._options.srcFolder,
-      absolute: true,
-      nosort: true,
-      nodir: true
-    });
+    const filePaths: string[] = await LegacyAdapters.convertCallbackToPromise(
+      glob,
```

Comment:
Ugh...

Reply:
Agreed. Let's do that in a separate change.